src/audio/PlaybackQueue.ts

```diff
@@ -132,6 +132,7 @@ export class PlaybackQueue {
                         // else no explicit next event, so find an event we haven't played that comes next. The live
                         // timeline is already most recent last, so we can iterate down that.
                         const timeline = arrayFastClone(this.room.getLiveTimeline().getEvents());
+                        const cliID = MatrixClientPeg.get().getUserId();
```

Comment:
You're right that this will result in better performance :+1: though let's call the variable something like `myUserId` or `ourUserId` please